terraform/panther_gcs_transport_type_infra/panther.tfvars

```diff
@@ -0,0 +1,21 @@
+# Basic Info
```

Comment:
I'm wondering if the UI code needs to be updated:

https://github.com/panther-labs/panther-enterprise/blob/5d43ef195a7037167f201fbedf55436f2072e6b7/web/src/pages/Configure/LogSources/GoogleCloudSources/GcpPubSub/GcpPubSubLogSourceWizard/SetupInfrastructure.tsx#L116

Although `production.tfvars` makes sense, as it contains the actual production settings and this is the sample?